### DISCUSSION TITLE
Is it possible to bypass the proxy on Asp.Net

### DISCUSSION THREAD

**Discussion by GrantByrn3** [scalar/scalar#3592] [Q&A]
Currently trying out Scalar documentation for my asp.net app.

It's not working on my company intranet because the proxy is outside the network where the api is. Scalar is running from the same app as the api so a proxy isn't needed for cors.

Is there a way to bypass the proxy?

**Comment by antlio** [comment]
hey @GrantByrn3, i believe this is ongoing in #3583

**Reply by GrantByrn3** [reply]
ok, yeah, that makes sense. Sounds like I'll have to wait a minute for that to get merged in and the accessible via the Asp.net nuget package.

Or, well... spin up my own proxy.

**Reply by marclave** [reply]
hey @GrantByrn3 this has been fixed in latest scalar release :)
